app/src/main/java/com/asfoundation/wallet/widget/BackupWarningView.java

```diff
@@ -52,11 +52,11 @@ private void init(@LayoutRes int layoutId) {
         }
       }
       break;
-      case R.id.later_action: {
-        if (onNegativeClickListener != null) {
-          onNegativeClickListener.onBackupClick(v, wallet);
-        }
-      }
+      //case R.id.later_action: {
```

Comment:
there is no action on negative feedback?
if not, remove commented code and unused code

Reply:
Yes, negative action was removed on the portrait layout and kept on the landscape.
I will remove the code.